src/AltinnCore/Runtime/RestControllers/DataController.cs

```diff
@@ -319,14 +325,51 @@ public class DataController : ControllerBase
             await serviceImplementation.RunServiceEvent(ServiceEventType.ValidateInstantiation);
```

Comment:
For me it does not seem correct to have validate instansiation on form. That is more a instance stuff. But we might have a instansiate form event.